1-src/api/auth/auth-utilities.mts

```diff
@@ -140,17 +140,9 @@ export const getUserLogin = async(email:string = '', password: string = ''):Prom
  Utility Methods
 ******************* */
 
-export const isRequestorAllowedProfile = async(clientProfile:USER, userProfile:USER):Promise<boolean> => {
-    if(clientProfile.userID === userProfile.userID || userProfile.isRole(RoleEnum.ADMIN)) return true;
-
-    //Test Member of Leader's Circle
-    if(userProfile.isRole(RoleEnum.CIRCLE_LEADER)) {
-        // const clientLeaderIDList:number[] = await DB_SELECT_CIRCLE_LEADER_IDS(clientProfile.userID);
-        // return (clientLeaderIDList.includes(userProfile.userID));
-    }
-    return false;
-}
-
+//Since request.jwtUserRole is max role; this utility tests if userRole is possible
+export const isMaxRoleGreaterThan = ({testUserRole, currentMaxUserRole}:{testUserRole:RoleEnum, currentMaxUserRole:RoleEnum}):boolean => 
```

Comment:
As it currently stands, this function doesn't return anything. Is that intentional?